newIDE/app/src/InstancesEditor/Background.js

```diff
@@ -15,6 +16,7 @@ export default class Background {
       width,
       height
     );
+    this._checkeredBackground.tint = 0x444444;
```

Comment:
Probably to do according to the theme

Reply:
Yes, but it's probably complicated and low priority. I only added this to save my eyes.

Reply:
Ok 👍